test/integration/generated/bodyComplexRest/README.md

```diff
@@ -0,0 +1,96 @@
+# BodyComplexRest client library for JavaScript
+
+Test Infrastructure for AutoRest
+
+**Please rely heavily on our [REST client docs](https://github.com/Azure/azure-sdk-for-js/blob/main/documentation/rest-clients.md) to use this library**
+
+Key links:
+
+- [Package (NPM)](https://www.npmjs.com/package/@msinternal/body-complex-rest)
+- [Samples](https://github.com/Azure-Samples/azure-samples-js-management)
+
+## Getting started
+
+### Currently supported environments
+
+- Node.js version 14.x.x or higher
+
+### Prerequisites
+
+- You must have an [Azure subscription](https://docs.microsoft.com/en-us/azure/cognitive-services/authentication?tabs=powershell#authenticate-with-an-authentication-token) to use this package.
+
+### Install the `@msinternal/body-complex-rest` package
+
+Install the BodyComplexRest client REST client library for JavaScript with `npm`:
+
+```bash
+npm install @msinternal/body-complex-rest
+```
+
+### Create and authenticate a `BodyComplexRestClient`
+
+To use an [Azure Active Directory (AAD) token credential](https://docs.microsoft.com/en-us/azure/cognitive-services/authentication?tabs=powershell#authenticate-with-an-authentication-token),
+provide an instance of the desired credential type obtained from the
+[@azure/identity](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/identity/identity#credentials) library.
+
+To authenticate with AAD, you must first `npm` install [`@azure/identity`](https://www.npmjs.com/package/@azure/identity) 
+
+After setup, you can choose which type of [credential](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/identity/identity#credentials) from `@azure/identity` to use.
+As an example, [DefaultAzureCredential](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/identity/identity#defaultazurecredential)
+can be used to authenticate the client:
+
+Set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables:
+AZURE_CLIENT_ID, AZURE_TENANT_ID, AZURE_CLIENT_SECRET
+
+Use the returned token credential to authenticate the client:
+
+```typescript
+import { BodyComplexRestClient } from "@msinternal/body-complex-rest";
+import { DefaultAzureCredential } from "@azure/identity";
+const client = BodyComplexRestClient(
+  "<ENDPOINT>",
+  new DefaultAzureCredential()
+);
+```
+
+## Key concepts
+
+### REST Client
+
+This client is one of our REST clients. We highly recommend you read how to use a REST client [here](https://github.com/Azure/azure-sdk-for-js/blob/main/documentation/rest-clients.md).
+
+## Examples
```

Comment:
I would also remove this examples section. They are already in the Documentation referenced above

Reply:
This is just a placeholder for the example. We plan to make this part smarter to enrich/refine here with the real and valid sample after we have samples generated.

Reply:
Removed